src/cli.rs

```diff
@@ -27,10 +27,10 @@ impl Args {
     }
 }
 
-pub fn args() -> Result<(Command, Args), String> {
+pub fn args(command_name: &str) -> Result<(Command, Args), String> {
     let mut args = env::args().skip(1);
-    if args.next() != Some("xbuild".into()) {
-        Err("must be invoked as cargo subcommand: `cargo xbuild`")?;
+    if args.next() != Some("x".to_string() + command_name) {
+        Err(format!("must be invoked as cargo subcommand: `cargo {}`", command_name))?;
```

Comment:
Shouldn't it be `cargo x{}` (with prepended `x`)?

Reply:
Yeah I missed that one.